bitnami/influxdb/README.md

```diff
@@ -51,18 +91,14 @@ To make this process easier, the chart contains the `resourcesPreset` values, wh
 
 ### Update credentials
```

Comment:
No way of managing the token via a Kubernetes secret? Something like a job or similar

Reply:
Impossible? No but it'd be very difficult given you have to take into account several things: was it already created? Are we using "memory" object store? Are we using "file" object store, if so, with N replicas?

Reply:
Addressed at https://github.com/bitnami/charts/pull/33743/commits/2a32a7e16d7456f515f04ffb17174a3467275f1b